src/Filter/ModelFilter.php

```diff
@@ -76,31 +77,20 @@ protected function handleMultiple(ProxyQueryInterface $query, string $alias, Fil
             return;
         }
 
-        $parameterName = $this->getNewParameterName($query);
+        $inExpression = $this->buildInExpression($query, $alias, $data);
 
         if ($data->isType(EqualOperatorType::TYPE_NOT_EQUAL)) {
-            $or = $query->getQueryBuilder()->expr()->orX();
-
-            $or->add($query->getQueryBuilder()->expr()->notIn($alias, ':'.$parameterName));
-
-            $associationMapping = $this->getAssociationMapping();
-
-            if (ClassMetadata::MANY_TO_MANY === $associationMapping['type']) {
-                $or->add(
-                    sprintf('%s.%s IS EMPTY', $this->getParentAlias($query, $alias), $this->getFieldName())
-                );
-            } else {
-                $or->add($query->getQueryBuilder()->expr()->isNull(
-                    sprintf('IDENTITY(%s.%s)', $this->getParentAlias($query, $alias), $this->getFieldName())
```

Comment:
We didn't had enough tests when this was added. It does not work for inverse relations. For example the UlidChildEntityAdmin filtering on the parent field. It is not related to the uuid, it simply does not work AFAIK.

@VincentLanglet do you found something like that on your previous projects?

Reply:
The error only happens when someone is trying to filter with the advanced filter "NOT EQUALS"

Reply:
I'm not sure I worked with the ModelFilter. I think I always use ChoiceFilter by selecting the id field.

Reply:
I think I found a solution, let's see